src/main/java/com/serenitydojo/Cat.java

```diff
@@ -0,0 +1,37 @@
+package com.serenitydojo;
+
+public class Cat extends Pet {
+//    private String name;
+    private String favroriteFood;
```

Comment:
typos in the variable and getter/setter
```suggestion
    private String favoriteFood;
```